common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Strategy.java

```diff
@@ -619,10 +620,26 @@ protected TokenResult getTokenResultFromHttpResponse(@NonNull final HttpResponse
                     tokenResponse.setCliTelemSubErrorCode(cliTelemInfo.getServerSubErrorCode());
                 }
             }
+            final String ccsRequestId = response.getHeaderValue(XMS_CCS_REQUEST_ID, 0);
+            if(null != ccsRequestId)
+            {
+                if(null != tokenResponse)
+                {
+                    Map<String, String> mapWithAdditionalEntry = new HashMap<String, String>();
+                    mapWithAdditionalEntry.put(XMS_CCS_REQUEST_ID, ccsRequestId);
+                    if(null != tokenResponse.getExtraParameters())
+                    {
+                        for(Map.Entry<String, String> entry : tokenResponse.getExtraParameters())
+                        {
+                            mapWithAdditionalEntry.put(entry.getKey(), entry.getValue());
+                        }
+                    }
+                    tokenResponse.setExtraParameters(mapWithAdditionalEntry.entrySet());
+                }
```

Comment:
The formatting / placement of the curly braces doesn't match with the format used in the rest of the codebase

Reply:
Thanks for catching this! I should have been more careful to look at existing style when making my changes, I just let muscle memory from working in C++ libs take over!

Fixed to use opening bracket on originating line of the new scope in latest commit